src/Symfony/Component/Validator/Tests/Resources/TranslationFilesTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Resources;
+
+class TranslationFilesTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        libxml_use_internal_errors(true);
+        libxml_clear_errors();
+    }
+
+    public function testXlfFilesAreValid()
+    {
+        $translationFiles = glob(__DIR__.'/../../Resources/translations/*.xlf');
+        foreach ($translationFiles as $filePath) {
```

Comment:
I think he meant a data provider, not a data collector. And this is indeed a good idea as a failure in a file would not prevent validating subsequent files (as they would be separate tests)